### PR TITLE
updated sign up db data

### PR DESCRIPTION
**Pull request by Edmund-Lui98**

### CODE REVIEW

File: lib/services/authentication.dart [hunk 1]
```diff
@@ -33,12 +32,15 @@ class Auth implements BaseAuth {
 
   @override
   Future<String> signUp(String email, String password) async {
+    print(email);
```
Comment (hoklaamc) [inline lib/services/authentication.dart:35]: print statements

File: lib/services/user.dart [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
-  Future updateUserData(String name, int age) async {
+  Future updateUserData(
+      String firstName, String lastName, String username, int bp) async {
     return await userCollection.document(uid).setData({
-      'name': name,
-      'age':age,
+      'firstName': firstName,
+      'lastName': lastName,
+      'username': username,
+      'bp': bp
```
Comment (hoklaamc) [inline lib/services/user.dart:17]: What's bp?

**Review by hoklaamc — APPROVED** [review]
Couple things but non blocking

Also did we want all those white space changes? Not sure if they are consistent

File: lib/services/authentication.dart (hunk 1 above)
Comment (Edmund-Lui98) [reply lib/services/authentication.dart:35]: oh yes those can be taken out... good catch

File: lib/services/user.dart (hunk 2 above)
Comment (Edmund-Lui98) [reply lib/services/user.dart:17]: bp = ball points
I think this is the term that Raymond used when he was talking about earning points through playing games

**Review by a6you — APPROVED** [review]

File: lib/services/user.dart (hunk 2 above)
Comment (hoklaamc) [reply lib/services/user.dart:17]: gotcha, let's rename that so it's clear, always avoid abbreviations for variable names if it's not too long

**Pull request merged**